resources/js/src/app/components/containers/ContainerItemList.js

```diff
@@ -76,6 +76,9 @@ Vue.component("container-item-list", {
                         items: (this.itemsPerPage > 1) ? 2 : 1
                     },
                     768: {
+                        items: (this.itemsPerPage > 2) ? 3 : ((items.itemsPerPage > 1) ? 2 : 1)
```

Comment:
Du verwendest einmal `this.itemsPerPage` und einmal `items.itemsPerPage`. Die Variable `items` existiert aber nicht. Generell lässt sich die Anweisung sicherlich vereinfachen:
`items: (this.itemsPerPage > 3) ? 3 : this.itemsPerPage`
(Sprich: wenn über `itemsPerPage` eigentlich mehr als 3 Artikel eingestellt sind, begrenze die Darstellung auf 3 Artikel, ansonsten nimm den eingestellten Wert.)

Reply:
Oh, das war wohl ein Typo 😅 ist mir beim Testen nicht aufgefallen, was wohl auch sehr für die Vereinfachung der Abfrage spricht. Werde ich ändern, sobald ich Zeit habe 👍 Danke für die Rückmeldung!